wit-manifest.json

```diff
@@ -260,12 +260,12 @@
         "source": "git@github.com:sifive/example-hca-metal.git"
     },
     {
-        "commit": "950b3ac73c3e5a2cf7e7463aa05b144e8616569c",
+        "commit": "ea10676cde77d8417b0b805b32631708f2f72a91",
```

Comment:
For both examples, add a comment the purpose of +16 for the address, and use metal api for cflush for consistency. you can put the asm in a comment as an alternate usage.

Reply:
updated examples with comment, metal API for cache flush.